x-pack/plugins/serverless/test/api_integration/services/index.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { services as xpackApiIntegrationServices } from '../../../../../test/api_integration/services';
```

Comment:
@pheyos I was wondering if it might cause the issue for TS, I don't think we can do imports like this into plugins.